src/err/err_state.rs

```diff
@@ -109,11 +109,22 @@ impl PyErrState {
         }))
     }
 
-    pub(crate) fn normalized(pvalue: &PyBaseException) -> Self {
+    pub(crate) fn lazy_bound(ptype: Bound<'_, PyAny>, args: impl PyErrArguments + 'static) -> Self {
```

Comment:
Was there a reason to add `lazy_bound` rather than just change `lazy`? As this is an internal function it'd be nice to migrate `lazy` over completely rather than duplicate, if possible.

Reply:
Mainly to avoid changing everything all at once.